packages/amplify-environment-parameters/src/environment-parameter-manager.ts

```diff
@@ -85,6 +85,49 @@ class EnvironmentParameterManager implements IEnvironmentParameterManager {
     return !!this.resourceParamManagers[getResourceKey(category, resource)];
   }
 
+  canBeClonedHeadlessly(): boolean {
```

Comment:
It might be better to return a reason for why an environment can't be cloned so we can display a better error message. Could the return value here be changed to
```ts
{ result: true } | {result: false, reason: string}
```
then build up a string of exactly what parameters are problematic in the case of false